script/tool/README.md

```diff
@@ -1,8 +1,60 @@
 # Flutter Plugin Tools
 
+Note: The commands in tools are designed to run under plugins/ or plugins/packages/.
+
 To run the tool:
 
 ```sh
 dart pub get
 dart run lib/src/main.dart <args>
 ```
+
+## Format Code
+
+```sh
+cd <path_to_plugins>/packages
+dart run ./script/tool/lib/src/main.dart format --plugins plugin_name
+```
+
+## Run static analyzer
+
+```sh
+cd <path_to_plugins>/packages
+pub run ./script/tool/lib/src/main.dart analyze --plugins plugin_name
+```
+
+## Run dart unit tests
+
+```sh
+cd <path_to_plugins>/packages
+pub run ./script/tool/lib/src/main.dart test --plugins plugin_name
+```
+
+## Run XCTests
+
+```sh
+cd <path_to_plugins>/packages
+dart run lib/src/main.dart xctest --target RunnerUITests --skip <plugins_to_skip>
+```
+
+## Publish and tag release
+
+``sh
+cd <path_to_plugins>/packages
+dart run ./script/tool/lib/src/main.dart publish-plugin --package $package
```

Comment:
what is `$package`?  did you mean `<package>`?

Reply:
yes, done